### ISSUE TITLE
Ivp with and without adjoint method.

### ISSUE THREAD

**Issue by zjowowen** [martenlienen/torchode#35]
Hi, I want to know if torchode always doing integration with adjoint method?

How about integration without adjoint method, so that no gradient is recorded?
I did not find information about it in the document. I wonder what's the best practice if without "AutoDiffAdjoint".
Or should I give a drift function where "with torch.no_grad()" used?

**Comment by martenlienen** [comment]
Hey, just wrap the solve to in `no_grad`, i.e.
```python
with torch.no_grad():
    sol = solver.solve(...)
```
